source/_drafts/Video-js-6-0-0-RC-0-The-first-RC.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Video.js 6.0.0-RC.0: The first Release Candidate'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - video.js 6.0
+---
+
+## Release Candiate Released
+Last week, we published the first Release Candidate (RC) of Video.js 6.0 on npm.
+This was a months-long team effort to make Video.js even better.
+However, to do so, we had to make a few breaking changes.
```

Comment:
I think it would be a good idea to link to the changelog here so that anyone interested can see what the breaking changes were. It may also be a good idea to specify that this is a major release because we needed to make breaking changes and we want to follow semantic versioning.